packages/ihe-gateway-sdk/src/models/patient-discovery.ts

```diff
@@ -19,12 +19,16 @@ export type PatientDiscoveryRequest = {
   xcpdGateways: XCPDGateways;
   samlAttributes: SamlAttributes;
   patientResource: Patient;
-  principalCareProviderIds?: NPIStringArray;
+  principalCareProviderNPIs?: NPIStringArray;
```

Comment:
Based on the schema this should be ids https://www.notion.so/metriport/XCPD-ITI-55-Interface-Outgoing-91b4e496b6b64ff6a041f55f3be5d1a5

Reply:
Oh weird.. At some point, I merged develop, and it changed it to `...NPIs` . I'll revert this 👍